### DISCUSSION TITLE
Get confidence score on StructuredDataClassifier

### DISCUSSION THREAD

**Discussion by theophiledebouetduportal** [keras-team/autokeras#1598] [Q&A] [answered]
When I run clf.predict() it only output the predicted score.
Is it possible get the confidence score on a StructuredDataClassifier model ? 

Thanks.

**Comment by haifeng-jin** [comment, marked as answer]
You would have to export the model to run `model.predict()`.

**Reply by theophiledebouetduportal** [reply]
thanks !
